pkg/minikube/driver/driver.go

```diff
@@ -22,10 +22,13 @@ import (
 	"sort"
 
 	"github.com/golang/glog"
+	"k8s.io/minikube/pkg/minikube/bootstrapper"
 	"k8s.io/minikube/pkg/minikube/registry"
 )
 
 const (
+	// Kubernetes in container using docker driver
+	KicDocker = "docker"
```

Comment:
my intent was finding it easier in code base, since docker is such a widely used word and hard to search for it. but I can see that it could cause more confusion for others. I will rename to docker.